docs/index.rst

```diff
@@ -22,9 +22,10 @@ Contents:
 .. toctree::
    :maxdepth: 2
 
-* :doc:`tutorial` 
-* :doc:`reference`
-* :doc:`howto`
+   tutorial
+   reference
+   howto
+   devel
```

Comment:
Btw, I moved these under the `.. toctree:: here assuming that was the intention. Also added `devel` because otherwise sphinx generates a warning that it's not part of any toctree. The resulting built docs are the same except, the bulleted list in the "Contents:" section of index.html is now generated automatically, including 2nd-level topics as per the `:maxdepth: 2` which I didn't change. Could change that to `:maxdepth: 1` to make it condensed as before.